README.md

```diff
@@ -103,6 +103,17 @@ inst\bin\lpython examples\expr2.py -o a.out
 a.out
 ```
 
+- After you update a test case file, you also need to update all the reference results associated with that test case:   
+
+```
+python run_tests.py -u --skip-run-with-dbg
+```
```

Comment:
The `--skip-run-with-dbg` flag should only be used on Windows, on Linux/macOS it should not be used.

Reply:
Yes, more specifically for the `MSVC` build setup. On windows with WSL, `./run_tests.py -u ` runs just fine.

Reply:
I see it's in the Windows section of the README. I guess that's fine. We should eventually refactor it into a dedicated section and show commands for all platforms.